kube_aws_autoscaler/main.py

```diff
@@ -398,15 +414,34 @@ def main():
     parser.add_argument('--enable-healthcheck-endpoint', help='Enable Healtcheck',
                         action='store_true')
     parser.add_argument('--no-scale-down', help='Disable scaling down', action='store_true')
+
     for resource in RESOURCES:
         parser.add_argument('--buffer-{}-percentage'.format(resource), type=float,
                             help='{} buffer %%'.format(resource.capitalize()),
                             default=os.getenv('BUFFER_{}_PERCENTAGE'.format(resource.upper()), DEFAULT_BUFFER_PERCENTAGE[resource]))
         parser.add_argument('--buffer-{}-fixed'.format(resource), type=str,
                             help='{} buffer (fixed amount)'.format(resource.capitalize()),
                             default=os.getenv('BUFFER_{}_FIXED'.format(resource.upper()), DEFAULT_BUFFER_FIXED[resource]))
+
+    parser.add_argument('--scale-down-step-fixed',
+                        help='Scale down strategy expressed in terms of instances count, defaults to 1. Note: value must be >= 1',
+                        type=int, default=os.getenv('SCALE_DOWN_STEP_FIXED', 1))
+    parser.add_argument('--scale-down-step-percentage',
+                        help='Scale down strategy expressed in terms of instances count, defaults to 0.01, i.e. 1%.',
```

Comment:
the default is 0.0, but the help says 0.01 :smirk:

Reply:
ah... nice catch.